crates/ruff/src/message/diff.rs

```diff
@@ -47,8 +47,14 @@ impl Display for Diff<'_> {
 
         let diff = TextDiff::from_lines(self.source_code.source_text(), &output);
 
-        writeln!(f, "{}", "ℹ Suggested fix".blue())?;
-
+        let message = match self.fix.applicability() {
+            Applicability::Automatic => "Fix",
+            Applicability::Suggested => "Suggested fix",
+            Applicability::Manual => "Possible fix",
+            _ => "Suggested fix",  // For backwards compatibility, unspecified fixes are 'suggested'
```

Comment:
I think we can change this case to explicitly cover only `Unspecified` if we drop the non-exhaustive from the enum https://github.com/charliermarsh/ruff/pull/4303/files#r1188177338